frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -29,7 +29,7 @@ export const saveButton = $('button[type=submit]');
 export const modalConfirmButton = $('#confirm-action');
 
 // YAML form
-export const alertManagerYamlForm = $('.co-alert-manager-yaml__form');
+export const alertManagerYamlEditor = $('.yaml-editor');
```

Comment:
I'd make this generic and move it into yaml.view.ts

Reply:
moved.